src/main/scala/org/clulab/timenorm/scate/TemporalNeuralParser.scala

```diff
@@ -249,7 +249,7 @@ class TemporalNeuralParser(modelStream: Option[InputStream] = None) extends Auto
     // for each batch, combine the different time operators into a single array
     for (i <- expandedTexts.indices.toArray) yield {
       val operators = nonOperators(i) ++ expOperators(i) ++ impOperators(i)
-      operators.sortBy { case (start, _, _) => start }
+      operators.sortBy { case (start, _, _) => start }.distinct
```

Comment:
There needs to be a test for the case that triggered this, i.e., make sure the test fails without this change and succeeds with it.